tests/ui/rfcs/rfc-2091-track-caller/error-with-naked.stderr

```diff
@@ -1,14 +1,20 @@
-error[E0736]: cannot use `#[track_caller]` with `#[naked]`
+error[E0736]: attribute incompatible with `#[naked]`
   --> $DIR/error-with-naked.rs:6:1
    |
 LL | #[track_caller]
-   | ^^^^^^^^^^^^^^^
+   | ^^^^^^^^^^^^^^^ this attribute is incompatible with `#[naked]`
+LL |
+LL | #[naked]
+   | -------- function marked with `#[naked]` here
 
-error[E0736]: cannot use `#[track_caller]` with `#[naked]`
+error[E0736]: attribute incompatible with `#[naked]`
   --> $DIR/error-with-naked.rs:18:5
    |
 LL |     #[track_caller]
-   |     ^^^^^^^^^^^^^^^
+   |     ^^^^^^^^^^^^^^^ this attribute is incompatible with `#[naked]`
```

Comment:
When macros are involved, relying just on the shown source snippet may sometimes not be enough to figure out which attribute is incompatible. Maybe keep the name of the attribute in either the error message or the label for the primary span?

Reply:
good call, the message is now "the `track_caller` attribute is incompatible with `#[naked]`"